docs/introduction.md

```diff
@@ -0,0 +1,33 @@
+---
+id: introduction
+---
+
+# Introduction
+
+## Features
+
+__SuperBench__ is a validation and profiling tool for AI infrastructure, which supports:
+
+* AI infrastructure validation and diagnosis
+  * Distributed validation tools to validate hundreds or thousands of servers automatically
+  * Consider both raw hardware and E2E model performance with ML workload patterns
+  * Build a contract to identify hardware issues
+  * Provide infrastructural-oriented criteria as Performance/Quality Gates for hardware and system release
+  * Provide detailed performance report and advanced analysis tool
+* AI workload benchmarking and profiling
+  * Provide comprehensive performance comparison between different existing hardware
+  * Provide insights for hardware and software co-design
+
+It includes micro-benchmark for primitive computation and communication benchmarking,
+and model-benchmark to measure domain-aware end-to-end deep learning workloads.
+
+:::note
+SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
+If you want to jump in early, you can try building latest code yourself.
+:::
```

Comment:
MD cannot know this semantics?

Reply:
it's not in [CommonMark](https://commonmark.org/), but one of the markdown extensions
e.g., syntax in https://docs.microsoft.com/en-us/contribute/markdown-reference